src/daemon/ca/certauth.rs

```diff
@@ -686,9 +776,7 @@ impl CertAuth {
         let issued =
             self.issue_child_certificate(&child, rcn.clone(), csr_info, limit, &config.issuance_timing, &signer)?;
 
```

Comment:
Since an explicit republication action was removed here, would it make sense to add a comment here to say that certs will be republished the next time the scheduled republication job runs?

Reply:
Or was this likely to be a no-op and was just adding noise to the event history?